### PR TITLE
Print sync status to cli as it syncs

### PR DESCRIPTION
**Pull request by SWvheerden**
## Description

This will print to the cli the current sync status as its busy syncing. 

## Motivation and Context

This will provide feedback to the user how far the node has synced and how much further still needs to be synced. 

See issue #1792 

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [x] I'm merging against the `development` branch.
* [x] I ran `cargo-fmt --all` before pushing.
* [x] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: base_layer/core/src/base_node/states/block_sync.rs [hunk 1]
```diff
@@ -263,7 +263,9 @@ async fn synchronize_blocks<B: BlockchainBackend + 'static>(
 
             info!(target: LOG_TARGET, "Synchronize missing blocks.");
             let mut height = sync_height;
+            println!("Starting to sync blocks");
             while height <= network_tip_height {
+                println!("[Syncing block {}/{}] \r", height, network_tip_height);
```
Comment (sdbondi) [inline base_layer/core/src/base_node/states/block_sync.rs:268]: Maybe a `trace!` level log?

**Review by sdbondi — COMMENTED** [review]
Adding terminal output inside core code is not a good idea and the state changes are already logged in the `next_event` function.  Adding a command that prints the current node state would be useful.

File: base_layer/core/src/base_node/states/block_sync.rs [hunk 2]
```diff
@@ -263,7 +263,9 @@ async fn synchronize_blocks<B: BlockchainBackend + 'static>(
 
             info!(target: LOG_TARGET, "Synchronize missing blocks.");
             let mut height = sync_height;
+            println!("Starting to sync blocks");
```
Comment (CjS77) [inline base_layer/core/src/base_node/states/block_sync.rs:266]: Remove / change to `debug!`

**Review by CjS77 — COMMENTED** [review]
I'm not a fan of this approach at all TBH. I know it's quick and dirty, but let's rather solve this issue properly with UI feedback coming via the CLI only.

**Comment by SWvheerden** [comment]
This is replaced with #1827

**Pull request closed without merging**